wonjong/app.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const logger = require("morgan");
+const dotenv = require("dotenv");
+const { DataSource } = require("typeorm");
+const { json } = require("express");
+dotenv.config();
+app.use(logger("combined"));
+require("dotenv").config();
+app.use(cors());
+app.use(express.json());
+const myDataSource = new DataSource({
```

Comment:
- myDataSource 보다는 `appDataSource` , `mysqlDataSource` 혹은 간결하게 `database`로 변수명을 변경해주세요. “my”라는 용어는 예제 코드에서는 사용할 수 있으나 실제 코드에서는 피해주세요.

```suggestion
app.use(express.json());

const appDataSource = new DataSource({
```